core/src/main/java/org/mule/runtime/core/processor/strategy/DefaultFlowProcessingStrategyFactory.java

```diff
@@ -29,39 +27,29 @@
 import java.util.function.Supplier;
 
 /**
- * Creates default processing strategy with same behavior as {@link ProactorProcessingStrategyFactory} apart from the fact it will
- * process synchronously without error when a transaction is active.
+ * Creates default processing strategy with same behavior as {@link WorkQueueProcessingStrategyFactory} apart from the fact it
+ * will process synchronously without error when a transaction is active.
  */
-public class DefaultFlowProcessingStrategyFactory extends ProactorProcessingStrategyFactory {
+public class DefaultFlowProcessingStrategyFactory extends WorkQueueProcessingStrategyFactory {
```

Comment:
can we design some way to make these changes through configuration rather than changing the code?